.gitignore

```diff
@@ -18,5 +18,6 @@ setuptools.egg-info
 .hg*
 .cache
 .idea/
+.vscode/
```

Comment:
Shouldn't `.idea` not be in there in that case?

It's also project-specific configs, not user specific. So far for setuptools I only had to configure `"ruff.organizeImports": false` because the rest of my user settings mostly align with the project's standards.

`.gitignore`-ing `.vscode` basically says "you're on your own if you use VSCode, we offer no support in terms of installing the right extensions and configuring them correctly for this project"

Anyway, I could try to do something similar to what you shared, but with local configs instead of something global.

Reply:
Even simpler I found `./.git/info/exclude` 😃 
![image](https://github.com/pypa/setuptools/assets/1350584/105059fb-57a3-4cec-b2fd-00a3ceffc1e8)

Reply:
> Shouldn't .idea/ (IntelliJ IDEs) not be in there in that case?

Probably... it slipped under the radar.